app/code/Magento/PageCache/Model/Varnish/VclGenerator.php

```diff
@@ -67,6 +74,7 @@ public function __construct(
         $accessList,
         $gracePeriod,
         $sslOffloadedHeader,
+        DirectoryList $directoryList,
```

Comment:
@torhoehn The new param should come as last one.

See: https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/
Add a new constructor param.

Reply:
You're absolutely right. I fixed that.

Reply:
@rodrigowebjump One of the test says, that arguments with default values must be at the end of the argument list. Should I undo my latest changes or add the DirectoryList as optional parameter?

Reply:
@rodrigowebjump I think everything looks fine now. :)